bakery/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.7-stretch
+
+LABEL maintainer="OpenStax Content Engineering"
+
+COPY ./src/scripts /code/scripts
+COPY ./requirements.txt /code
+WORKDIR /code/
+
+RUN pip install -r requirements.txt
```

Comment:
Since this is more of a dependency of the CLI rather than a containerization of the CLI itself, I wonder if it should live at the top level of this directory, rather than inside the scripts directory. For the same reason, I feel `output-producer-bakery` may not be the best name for it, maybe `cops-bakery-scripts`?

Reply:
@tomjw64 - This is a good point and I debated that myself. I ended up pushing the PR with it this way, but placing the Dockerfile and `requirements.txt` in `bakery/src/scripts` was the other option I liked too. Thanks for that feedback!

Also, ack on the corresponding naming convention adjustment for the image.